docs/java/usage.asciidoc

```diff
@@ -0,0 +1,218 @@
+== Getting started
+
+=== Maven Repository
+
+The low-level Java REST client is hosted on
+http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.elasticsearch.client%22[Maven
+Central]. The minimum Java version required is `1.7`.
+
+Here is how you can configure the dependency using maven as a dependency manager.
+Add the following to your `pom.xml` file:
+
+[source,xml]
+--------------------------------------------------
+<dependency>
+    <groupId>org.elasticsearch.client</groupId>
+    <artifactId>rest</artifactId>
+    <version>${es.version}</version>
```

Comment:
You say it is compatible with all versions of elasticsearch, so I'd imagine users replacing {es.version} with 1.7. Perhaps add a callout saying that this is the version of the ES client, starting with 5.0.0